WeWatch/WeWatch/Preview Content/Assets/Color.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Color.swift
+//  WeWatch
+//
+//  Created by Anton on 25/11/2024.
+//
+
+import SwiftUI
+
+extension Color {
+    static var fireyRed = Color("fireyRed")
+    static var blackColor = Color("blackColor")
+    static var darkGreyColor = Color("darkGreyColor")
+    static var lightGreyColor = Color("lightGreyColor")
+    static var whiteColor = Color("whiteColor")
```

Comment:
I prefer adding access modifiers for each item inside the extension. In this case, use `internal`. If you don't remember about access modifiers: https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol/#Access-Levels